src/ui/server/index.ts

```diff
@@ -35,6 +36,15 @@ app.use(seo);
 app.use(security);
 app.use(compression());
 
+const limiter = rateLimit({
+  windowMs: 1 * 60 * 1000, // 1 minute
+  max: 1000, // 1K requests / 1 window
```

Comment:
can we put this and above in constants?

Reply:
Could do - I think we should do everything in the UI server in one go though - can be future improvement? There's other instances (probably the same in DTFS), e.g for cookies, csrf etc